integration-tests/bats/system-tables.bats

```diff
@@ -137,6 +143,25 @@ teardown() {
     [[ "$output" =~ "create-table-branch" ]] || false
 }
 
+@test "system-tables: query dolt_remotes system table" {
+    run dolt sql -q "select count(*) from dolt_remotes" -r csv
+    [ $status -eq 0 ]
+    [[ "$output" =~ 0 ]] || false
+
+    mkdir remote
+    dolt remote add origin file://remote/
+
+    run dolt sql -q "select count(*) from dolt_remotes"
+    [ $status -eq 0 ]
+    [[ "$output" =~ 1 ]] || false
+
+    regex='file://.*/remote'
+    run dolt sql -q "select * from dolt_remotes" -r csv
+    [ $status -eq 0 ]
+    [[ "${lines[0]}" = name,url,fetch_specs,params ]] || false
+    [[ "${lines[1]}" =~ origin,file://.*/remote,[refs/heads/*:refs/remotes/origin/*,map[] ]] || false
```

Comment:
@zachmu keep fetch specs and params? reformat the string output? drop one or both?